tests/Hooks/Migration/Unit/MigratesHookTest.php

```diff
@@ -29,6 +29,7 @@ class MigratesHookTest extends TestCase
 
     protected function afterSetUp()
     {
+        resolve('migrator')->path(__DIR__);
```

Comment:
I think it would have been better to also include a test through a ServiceProvider (the approach that packages probably take)

Reply:
I was attempting to keep the test relatively small in scope (i.e. don't test too many things at once or you won't know what actually broke).  the Provider::loadMigrationsFrom() simply passes that path to the method I call here

Reply:
Can understand, with that thought: Let's fix this in a later PR and put this as something to do on our Project Board :)